### DISCUSSION TITLE
kubernetes-handbook/networking/cilium/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#101] [Announcements]
# kubernetes-handbook/networking/cilium/

Cilium 是一款开源软件，也是 CNCF 的孵化项目，目前已有公司提供商业化支持，还有基于 Cilium 实现的服务网格解决方案。最初它仅是作为一个 Kubernetes 网络组件。Cilium 在 1.7 版本后推出并开源了 Hubble，它是专门为网络可视化设计

https://lib.jimmysong.io/kubernetes-handbook/networking/cilium/

**Comment by shaofuzhang** [comment]
eBPF 简史 - ibm.com https://www.ibm.com/developerworks/cn/linux/l-lo-eBPF-history/index.html
链接地址无了

**Reply by rootsongjc** [reply]
看来是原链接被删除了，我把文章里的这个链接也删了吧。
